json-rpc/src/tests/unit_tests.rs

```diff
@@ -469,6 +486,15 @@ fn test_get_account_transaction_impl(
             let data: Option<TransactionView> = fetch_data(resp);
             let tx_view = data.expect("Transaction didn't exists!");
 
+            let expected_tx = mock_db
+                .all_txns
+                .iter()
+                .filter(|x| x.as_signed_user_txn().is_ok())
+                .map(|x| x.as_signed_user_txn().unwrap())
+                .find(|x| x.sender() == *acc && x.sequence_number() == seq)
```

Comment:
nit, can ignore: I feel this can be more concise

Reply:
fixed